src/pipx/venv.py

```diff
@@ -242,7 +242,7 @@ def install_package_no_deps(self, package_or_url: str, pip_args: List[str]) -> s
     def get_venv_metadata_for_package(self, package: str) -> VenvMetadata:
         data = json.loads(
             get_script_output(
-                self.python_path, VENV_METADATA_INSPECTOR, package, str(self.bin_path)
+                self.python_path, VENV_METADATA_INSPECTOR, package, self.bin_path
```

Comment:
I always cast to string because we've run into problems on windows when passing `Path` type arguments and assuming they will be auto-cast to strings. The CI tests passed so I guess this is okay.

Reply:
Inside of `run_subprocess()`, line 112 manually and explicitly casts every item in the command list to a string.  This was another convenience of `run_subprocess()`.

Reply:
I guess now it's line 111 of `util.py`:
```
cmd_str_list = [str(c) for c in cmd]
```